src/app/features/ui/searchbar-ui/searchbar/searchbar.component.ts

```diff
@@ -0,0 +1,56 @@
+import { NgClass } from '@angular/common';
+import { Input, Output, EventEmitter } from '@angular/core';
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { MatIconModule } from '@angular/material/icon';
+import { SearchResult } from '../search.types';
+import { SearchbarDropdownComponent } from '../searchbar-dropdown/searchbar-dropdown.component';
+
+@Component({
+  selector: 'app-searchbar',
+  standalone: true,
+  imports: [SearchbarDropdownComponent, MatIconModule, NgClass],
+  templateUrl: './searchbar.component.html',
+  styleUrls: ['./searchbar.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class SearchbarComponent {
+  @Input() typeOfEvent = true;
+  @Input() searchResults: SearchResult[] = [];
+  @Output() newSearchPhraseEvent = new EventEmitter<string>();
+  @Output() newSearchTypeEvent = new EventEmitter<string>();
+
+  searchPhrase = '';
+  defaultSearchType = 'tekst';
+  selectedSearchType = this.defaultSearchType;
+
+  isEmpty = true;
+  isDropdownNotVisible = true;
+  minNumberOfLettersToShowDropdown = 1;
+  maxNumberOfResultsToDisplay = 6;
```

Comment:
dodajmy przed nimi `readonly` bo z tego co widzę to są stałe i nie zmieniają się wczasie, dodatkowo jeśli to są stałe możemy przyjąć zapis zmiennej `MIN_NUMBER_OF_LETTERS_TO_SHOW_DROPDOWN`

Reply:
Stałe zostały zmienione na readonly. Nazwy stałych również zostały zmienione na duże litery zgodnie z sugerowaną konwencją.